src/tests/test_magic_cmd.py

```diff
@@ -391,6 +391,63 @@ def test_table_profile_store(ip, tmp_empty):
     assert report.is_file()
 
 
+def test_table_profile_with_snippets(ip, tmp_empty):
```

Comment:
it's challenging to verify `%sqlcmd explore` but we can at leat check that the command runs without errors: https://jupysql.ploomber.io/en/latest/community/developer-guide.html#unit-testing

and maybe we can grab the output? (even if we cannot click on the button for pagination)

Reply:
We are testing the data fetch functionality for `%sqlcmd explore` [here](https://github.com/AnirudhVIyer/jupysql/blob/658-snippet-support-sqlcmd/src/tests/test_util.py#L320). It looks like they are working with the new change. Maybe we can add these functions to the integration tests also.

Reply:
Added a test `test_table_explore_with_snippets`